src/lib.rs

```diff
@@ -443,12 +443,12 @@ impl<'a> EbpfVm<'a> {
         let mut pc: usize = entry;
         self.last_insn_count = 0;
         while pc * ebpf::INSN_SIZE < prog.len() {
-            trace!("    BPF: {:5?} {:016x?} frame {:?} pc {:4?} {}",
-                   self.last_insn_count,
-                   reg,
-                   frames.get_frame_index(),
-                   pc + ebpf::ELF_INSN_DUMP_OFFSET,
-                   disassembler::to_insn_vec(&prog[pc * ebpf::INSN_SIZE..])[0].desc);
+            // trace!("    BPF: {:5?} {:016x?} frame {:?} pc {:4?} {}",
```

Comment:
`if log::log_enabled!(log::Level::Trace) { ...` perhaps?

Reply:
lol, what a workaround.  I'll try that but I think fundamentally the logging crates are not behaving ideally...

Reply:
@mviines Doesn't help :-(

This also implies that if we turn on logging for any module at some level then all the messages system-wide for that level are formatted.  And the formatting is expensive.